.github/pull_request_template.md

```diff
@@ -1,6 +1,14 @@
 <!--
+##
+####         Note: the Lilac master branch has been created.  Please consider whether your change
+    ####     should also be applied to Lilac.  If so, make another pull request against the
+####         open-release/lilac.master branch, or ping @nedbat for help or questions.
+##
+
 Please give the pull request a short but descriptive title.
-Use [conventional commits](https://www.conventionalcommits.org/) to separate and summarize commits logically.
+Use conventional commits (https://www.conventionalcommits.org/ and
+https://open-edx-proposals.readthedocs.io/en/latest/oep-0051-bp-conventional-commits.html)
```

Comment:
Why link to both? Does the OEP not cover enough details?

Reply:
Good point, and even if the OEP doesn't, it links to the other.  I've reduced it to:
```
Please give the pull request a short but descriptive title.
Use conventional commits to separate and summarize commits logically:
https://open-edx-proposals.readthedocs.io/en/latest/oep-0051-bp-conventional-commits.html
```